pkg/sessions/persistence/ticket_test.go

```diff
@@ -43,8 +43,8 @@ var _ = Describe("Session Ticket Tests", func() {
 						Name: "dummy",
 					},
 				},
-				encodedTicket: fmt.Sprintf("%s.%s",
-					"dummy-0123456789abcdef",
+				encodedTicket: fmt.Sprintf("v2.%s.%s",
```

Comment:
Instead of replacing this test with the new version please extend all tests cases by adding each test once more using the new version. I know it might be a bit redundant but this way we guarantee a full test coverage for both versions.